app/controllers/api/v1/code_schools_controller.rb

```diff
@@ -2,7 +2,50 @@ module Api
   module V1
     class CodeSchoolsController < ApplicationController
       def index
-        render json: CodeSchools.all
+        render json: CodeSchool.all
+      end
+      
+      def create 
+        @school = CodeSchool.new(params_school)
+        
+        if @school.save 
+          render json: @school
+        else 
+          render json: { :errors => @school.errors.full_messages }
+        end
+      end
+      
+      def show 
+        @school = CodeSchool.where(id: params[:id]).first
+        if @school
+          render json: @school
+        else
+          render json: { error: 'No such record' }, status: :not_found
+        end
+      end
+      
+      def update 
+        @school = CodeSchool.find(params[:id])
+        @school.update(params_school)
+        if @school.save
+          render json: @school
+        else 
+          render json: { :errors => @school.errors.full_messages }
+        end
+      end
+      
+      def destroy
```

Comment:
Please be sure to update our API documentation with these new endpoints (`index, create, show, update, destroy`).  

You'll do so by updating the  [apiary.apib](https://github.com/OperationCode/operationcode_backend/blob/master/apiary.apib) file, following the pattern in the `TeamMember` section.

Here is what that file yields:  http://docs.operationcodeapi.apiary.io/#